engines/native/engine.go

```diff
@@ -32,6 +32,10 @@ func (engineProvider) ConfigSchema() schematypes.Schema {
 
 func (engineProvider) NewEngine(options engines.EngineOptions) (engines.Engine, error) {
 	var c config
+
+	// apply default values
+	c.CreateUser = true
+
```

Comment:
I have the feeling that `MustMap` will override it. As my suggestion to enforce explicitly configuration of this item, this is irrelevant anyway.